.github/workflows/ci.yml

```diff
@@ -66,6 +66,9 @@ jobs:
             kubectl get redisenterpriseclusters.app.redislabs.com test-cluster --output jsonpath='{.status}{"\n"}'
             sleep 5;
           done
+          echo "waiting loop has finished."
+          sleep 5;
+          kubectl get redisenterpriseclusters.app.redislabs.com test-cluster --output jsonpath='{.status}{"\n"}'
```

Comment:
This check on the status is to confirm the final state from the step.  I noticed that the step would complete, but the last output would not be what I expected during an investigation.  The same type of confirmation output has also be added to the Install database step below.